.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,47 @@
+## Summary
+
+Please describe your changes and what you plan to introduce as part of this pull request submissions. If it fixes a bug or resolves a feature request, be sure to link to the issue numbers following the template below.
+
+## Pull Request Checklist
+🚨Please review the [guidelines for contributing](https://vmweventbroker.io/community) to this repository.
+
+- [ ] Please ensure that you are making a pull request against the **Development** branch
+- [ ] Please use the `WIP` keyword in the title of your PR if you are not ready for review
+- [ ] Please ensure that you have opened a Github Issue if you are resolving/fixing a problem
+- [ ] Please ensure that you have [signed](https://help.github.com/en/github/authenticating-to-github/signing-commits) all commits and that you have [squashed](https://medium.com/@slamflipstrom/a-beginners-guide-to-squashing-commits-with-git-rebase-8185cf6e62ec) all relevant commmits related to your change
+- [ ] Please make sure that you have tested your change locally by successfully [building and deploying the VMware Event Broker Appliance](https://vmweventbroker.io/kb/contribute-appliance) and/or [building and deploying VMware Event Router](https://vmweventbroker.io/kb/contribute-eventrouter)
+- [ ] Please include any relevant screenshots and/or output as part of your testing
+- [ ] Please include any documentation updates that is applicable for your changes
+
+## Change Type
+
+What types of changes does your code introduce to the VMware Event Broker Appliance?
+
+_Put an `x` in all boxes that apply_
+
+Please check the type of change your PR introduces:
+- [ ] Bugfix (non-breaking change which fixes an issue)
+- [ ] Feature (non-breaking change which adds functionality)
+- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
+- [ ] Code style update (formatting, renaming)
+- [ ] Refactoring (no functional changes, no api changes)
+- [ ] Build related changes
+- [ ] Documentation changes
+- [ ] Other (please describe):
+
+## Resolved Issues
+
+* Issue Numbers:
```

Comment:
Does this imply closing/resolving once merged or do we need to add this specifically in the template to simplify this?

Reply:
It can ... I was thinking mainly as a reference point but if it does resolve an Issue, we should simply use this section for that. Is there something else we should add to make this clearer?

Reply:
Oh, so you're saying it will automatically close the associated issues with the current proposal (i.e. no need for explicit "closes" or "resolves" in that section)?

Reply:
No, we'll need to use the key word `closes: #<id>` for it to automatically close. I've added a bit more details + example so this is more clear. Let me know if the new update looks good?

Reply:
Looks better. Since 99% of PRs will likely only refer to one issue, I think having one line is sufficient-plus some information in the text like "add multiple Closes X lines depending on the number of issues resolved".

Reply:
Done!